load_balancer_test.go

```diff
@@ -0,0 +1,177 @@
+func TestLoadBalancerHandler_AttachedInstances(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v1/loadbalancer/instance_list", func(writer http.ResponseWriter, request *http.Request) {
+		response := `{"instance_list": ["1234", "2341"]}`
+		fmt.Fprintf(writer, response)
+	})
+
+	instanceList, err := client.LoadBalancer.AttachedInstances(ctx, 12345)
+
+	if err != nil {
+		t.Errorf("LoadBalancer.AttachedInstances returned %+v, ", err)
+	}
+
+	expected := &InstanceList{InstanceList: []string{"1234", "2341"}}
+
+	if !reflect.DeepEqual(instanceList, expected) {
+		t.Errorf("LoadBalancer.AttachedInstances returned %+v, expected %+v", instanceList, expected)
+	}
+}
+
+func TestLoadBalancerHandler_AttachInstance(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v1/loadbalancer/instance_attach", func(writer http.ResponseWriter, request *http.Request) {
+		fmt.Fprint(writer)
+	})
+
+	err := client.LoadBalancer.AttachInstance(ctx, 12345, 45678)
+
+	if err != nil {
+		t.Errorf("LoadBalancer.AttachInstance returned %+v, ", err)
+	}
+}
+
+func TestLoadBalancerHandler_DetachInstance(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v1/loadbalancer/instance_detach", func(writer http.ResponseWriter, request *http.Request) {
+		fmt.Fprint(writer)
+	})
+
+	err := client.LoadBalancer.DetachInstance(ctx, 12345, 45678)
+
+	if err != nil {
+		t.Errorf("LoadBalancer.DetachInstance returned %+v, ", err)
+	}
+}
+
+func TestLoadBalancerHandler_GetHealthCheck(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v1/loadbalancer/health_check_info", func(writer http.ResponseWriter, request *http.Request) {
+		response := `{ "protocol": "http","port": 81,"path": "/test","check_interval": 10,"response_timeout": 45,"unhealthy_threshold": 1,"healthy_threshold": 2}`
+		fmt.Fprintf(writer, response)
+	})
+
+	health, err := client.LoadBalancer.GetHealthCheck(ctx, 12345)
+
+	if err != nil {
+		t.Errorf("LoadBalancer.GetHealthCheck returned %+v, ", err)
+	}
+
+	expected := &HealthCheck{
+		Protocol:           "http",
+		Port:               81,
+		Path:               "/test",
+		CheckInterval:      10,
+		ResponseTimeout:    45,
+		UnhealthyThreshold: 1,
+		HealthyThreshold:   2,
+	}
+
+	if !reflect.DeepEqual(health, expected) {
+		t.Errorf("LoadBalancer.GetHealthCheck returned %+v, expected %+v", health, expected)
+	}
+}
+
+func TestLoadBalancerHandler_SetHealthCheck(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v1/loadbalancer/health_check_update", func(writer http.ResponseWriter, request *http.Request) {
+		fmt.Fprint(writer)
+	})
+
+	health := &HealthCheck{
+		Protocol:           "HTTPS",
+		Port:               8080,
+		Path:               "/health",
+		CheckInterval:      4,
+		ResponseTimeout:    5,
+		UnhealthyThreshold: 3,
+		HealthyThreshold:   4,
+	}
+	err := client.LoadBalancer.SetHealthCheck(ctx, 12345, health)
+
+	if err != nil {
+		t.Errorf("LoadBalancer.SetHealthCheck returned %+v, ", err)
```

Comment:
do we want to return an expected value? if not, we can remove the comma after %+v